### ISSUE TITLE
Cannot Build SoapUI 5.6.0-SNAPSHOT

### ISSUE THREAD

**Issue by FrankOsn** [SmartBear/soapui#478]
Could not resolve dependencies for project com.smartbear.soapui:soapui:jar:5.6.0-SNAPSHOT: The following artifacts could not be resolved: com.smartbear.utils.analytics:analytics-core:jar:5.6.0-SNAPSHOT, com.smartbear.utils.analytics:out-app-analytics-provider:jar:5.6.0-SNAPSHOT: Failure to find com.smartbear.utils.analytics:analytics-core:jar:5.6.0-SNAPSHOT in https://www.soapui.org/repository/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of smartbear-soapui-repository has elapsed or updates are forced
![image](https://user-images.githubusercontent.com/9128023/65443276-669f0600-ddf3-11e9-9ca1-d79c1122c6da.png)

**Comment by IlyaAvdeev** [comment]
@FrankOsn please pull the latest changes from "next" branch and try again. Let me know the results. Good luck!

**Comment by FrankOsn** [comment]
Only compiling on neatbeans give me a success build. When trying to call via mvn, both windows and linux fail.

![image](https://user-images.githubusercontent.com/9128023/65479546-764e3700-de53-11e9-9dd5-687b4a672bb9.png)

I'm attaching the error Log from windows env. As for linux, I think it's more like a maven plugin error. I'll dig in a bit more and share my results.

[WIN_Error_Log.zip](https://github.com/SmartBear/soapui/files/3645102/WIN_Error_Log.zip)

**Comment by IlyaAvdeev** [comment]
did you do `mvn clean install`?
works for me on Win

**Comment by FrankOsn** [comment]
Yup. Did it. I’ll check for the maven installation. Maybe I miss something. 
Aside that, the only way to run it was on netbeans win env.

**Comment by IlyaAvdeev** [comment]
are you sure there is no additional `{user.profile}\.m2\settings.xml`?

**Comment by stefv** [comment]
Hi.

I'm using the 5.7.0-SNAPSHOT (master of soapui) and I can confirm these snapshots don't exist in https://rapi.tools.ops.smartbear.io/nexus/content/groups/public/com/smartbear/utils/analytics/analytics-core/ repository. Maybe you have a batch cleaning the snapshots after a delay ?

```
<analytics.version>5.6.0-SNAPSHOT</analytics.version>

 <dependency>
            <groupId>com.smartbear.utils.analytics</groupId>
            <artifactId>analytics-core</artifactId>
            <version>${analytics.version}</version>
</dependency>

<dependency>
            <groupId>com.smartbear.utils.analytics</groupId>
            <artifactId>out-app-analytics-provider</artifactId>
            <version>${analytics.version}</version>
</dependency>
```

The last version at this date is the 5.5.1.

If you don't have changes in the analytics, can you please leave the release version in the pom ?
If you have changes, can you please release the snapshots or publish the repo on Github ?

Thank you.
